08-location.Rmd

```diff
@@ -96,7 +96,7 @@ input = mutate_all(input, funs(ifelse(. %in% c(-1, -9), NA, .)))
 After the data preprocessing, we convert the table into a raster stack.
 First, we use the x- and y-coordinates to convert the `data.frame` into a `SpatialPolygonsDataFrame` (we do not use sf-objects here, since the **raster** package does not support `sf`).
 <!-- make sure that this is the case -->
-`gridded()` converts `SpatialPolygonsDataFrame` into a `SpatialGridDataFrame` which in turn can be used as an input for **raster**'s `stack()` function.
+`gridded()` converts a `SpatialPolygonsDataFrame` into a `SpatialGridDataFrame` which in turn can be used as an input for **raster**'s `stack()` function.
```

Comment:
Note for later: we should use `sf` objects when this becomes possible.

Reply:
Sure if **raster** ever gets updated...